frontend/app-preview/src/views/LandingPage.tsx

```diff
@@ -72,7 +73,7 @@ export const LandingPage = ({ variant = 'preview' }: LandingPageProps) => {
             app={app}
             user={user}
             repository={repository}
-            buttonActions={appPreviewButtonActions(org, app, selectedLayoutInEditor)}
+            buttonActions={appPreviewButtonActions(org, app, instanceId)}
```

Comment:
The user was re-directed to `layout=null` when the `layoutName` was the argument, we need to pass the `instanceId` as an argument here instead. The reason for this is that we are reading the current `layoutName` from local storage when the user clicks on the "back to ux-editor" button. 

Another issue with passing the `layoutName` as an argument is that the URL will not be updated to "Side2" when the user navigates within the preview-modus.